webapp/reviews/models.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+from django.db import models
+
+class Review(models.Model):
+    book = models.ForeignKey(
+        'book_details.Book',
+        related_name='reviews')
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name='reviews')
+    anonymous = models.BooleanField(
+        'Anonymous')
+    rating = models.IntegerField(
+        'Rating',
+        null=True,
+        blank=True,
+        choices=[
+            (1, '1'),
+            (2, '2'),
+            (3, '3'),
+            (4, '4'),
+            (5, '5')
+        ])
+    comment = models.TextField(
+        'Comment',
+        max_length=500,
+        blank=True)
+
+    class Meta:
```

Comment:
Could you explain what this does?

Reply:
https://docs.djangoproject.com/en/1.11/ref/models/options/#unique-together

It puts a unique constraint on the combination `('book', 'user')`, since the same user should not be able to have multiple reviews on the same book.

Reply:
Ah, I see. Good catch, thank you.